src/_base/helm/app/templates/application/webapp/ingress-istio.yaml

```diff
@@ -11,6 +11,11 @@ spec:
  - {{ index .Values.docker.services "php-base" "environment" "APP_HOST" }}
  gateways:
  - istio-system/{{ .Release.Namespace }}-gateway
+{{- if index . "Values.istio.additionalGateways" }}
```

Comment:
I don't think index traverses dots

Reply:
well it does, but I've only seen it do that when doing `index .Values "istio" "additionalGateways", but that wont check if .Values.istio exists so would fail too

Reply:
hmm

Reply:
how about using with combined with if index

"with skips the code block if the variable is absent from the template"

Reply:
or alternatively, ensure .Values.istio[.additionalGateways] = [] is defined in the base config

Reply:
yes latter is cleaner